_posts/2016-10-12-ome-files-cpp-0.2.2.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: Release of OME Files C++ 0.2.2
+intro-blurb: The OME team is pleased to announce the release of OME Files C++ 0.2.2
+---
+
+Today we are releasing OME Files C++ 0.2.2.
+
+OME Files is a reference implementation of the OME Data Model and OME-TIFF
+file format for the storage and interchange of biological imaging data and
+metadata.  It provides support for:
+
+-   The OME Data Model (model objects, metadatastore, XSL transforms and XML
+    validation)
+-   OME-TIFF reader
+-   OME-TIFF writer
+
+The major focus of this release is to extend the support for features in older
+OME-XML model versions and to improve the support for automated testing all
+OME-XML and OME-TIFF features.  This will continue for the following release.
+
+Major changes in this release include:
+
+-   OME-TIFF reader and writer updates
+
+**Downloads**
+
+The software release is available at 
+[http://downloads.openmicroscopy.org/ome-files-cpp/0.2.2/](http://downloads.openmicroscopy.org/ome-files-cpp/0.2.2/).
```

Comment:
Should be [archived downloads] cf https://github.com/openmicroscopy/www.openmicroscopy.org/pull/53/files#diff-639b1eb8b9357fce496f08267c893ef8R36